linode/manager
# M3-5590: Storybook for Modals

## Description

> ⚠️ Work in progress

Updates Modals in Storybook and use `mdx`

## How to test

- Check **Storybook**
- Check for regressions in Modal components
